docs/concepts/index.md

```diff
@@ -21,10 +21,10 @@ search:
 | __[Schedules](/concepts/schedules)__               | Tell the Prefect API how to create new flow runs for you automatically on a specified cadence.     |
 | __[Results](/concepts/results)__                   | The data returned by a flow or a task.                                 |
 | __[Artifacts](/concepts/artifacts)__               | Formatted outputs rendered in the Prefect UI, such as markdown, tables, or links.  
-| __[Incidents](/concepts/incidents)__               | Identify, rectify and document issues in mission-critical workflows.       |
 | __[States](/concepts/states)__                     | Rich objects that capture the status of a particular task run or flow run.                                                                            |
 | __[Blocks](/concepts/blocks)__                     | Prefect primitives that enable the storage of configuration and provide a UI interface.          |
 | __[Task Runners](/concepts/task-runners)__         | Configure how tasks are run - concurrently, in parallel, or in a distributed environment. |
+| __[Events](/concepts/events)__                     | Notifications of changes that provide observability and can act as automation triggers.         |
```

Comment:
I think I'm missing context on what's happening :)

Reply:
Sorry, I should have put in the description.

Currently, Events is in the Concepts nav bar section and Incidents is in the Cloud navbar section.
On the Concepts index page Incidents was referenced and Events was not. 

This PR rectifies the problem by removing Incidents from the index page and adding Events.

Reply:
oh I see ! but shouldn't it actually be the case that neither should be there and both should be in the cloud toc and nav bar?

Reply:
I think it's a gray area for events. @billpalombi laid out some nice criteria for what fits in the Concepts section of the docs:

- If you were explaining how Prefect works, would you need to use this word?
- If there are a diagram of Prefect components, would this word be on it?
- Does this word have unique meaning in Prefect relative to other systems?

I think the answers to all three are it depends/maybe/somewhat. :) 

I could lean toward moving Events  to the Cloud section.

Reply:
i'd argue if it's a cloud-only feature, then it shouldn't be necessary to explain how prefect works, both of which applies to events and incidents

Reply:
With Automations in Concepts, maybe best to keep Events there too, as they should probably stick together. I could merge this PR and then open another if we decided to move other items to Cloud.

Reply:
Incidents is the only cloud feature that ended up in the Prefect concepts, so my vote would be to just remove Incidents from there. Happy to defer to others on that.

Reply:
@serinamarie do I understand correctly that you are saying that this PR is good as it stands?

Reply:
I think @serinamarie is saying that incidents being under the Cloud section only makes sense if other cloud-exclusive features are there too. I do think it's a little confusing right now because it's hard to differentiate what's a Prefect "concept" and what's a feature of Cloud, despite the badges. It doesn't feel quite right to me that the Concepts section keeps growing every time we add a cloud-exclusive capability.

Reply:
yeah, I'm saying this PR could probably just remove incidents from the concepts page and add events to the cloud page

Reply:
I agree with @serinamarie

Reply:
Thank you all. I've moved events to cloud.